src/vendor/vendor.c

```diff
@@ -1834,39 +1809,46 @@ int DiscoverDM_ForAllComponents(char *objs_filename, char *params_filename)
 ** \return  USP_ERR_OK if successful
 **
 **************************************************************************/
-int DiscoverDM_ForComponent(rdk_component_t *rdkc, kv_vector_t *rdk_objects, kv_vector_t *rdk_params)
+int DiscoverDM_ForComponent(kv_vector_t *rdk_objects, kv_vector_t *rdk_params)
 {
-    int i;
-    int ccsp_err;
+    int rbus_err;
     int err;
-    parameterInfoStruct_t **param_infos = NULL;
-    parameterInfoStruct_t *param_info;
-    int num_param_infos = 0;
     char *write_status;
+    rbusElementInfo_t* elem;
+    const char *component, *c1;
 
     // Exit if unable to determine all instances provided by this rdk component
-    USP_LOG_Info("%s: Getting parameters and objects for '%s'", __FUNCTION__, rdkc->group_name);
-    ccsp_err = CcspBaseIf_getParameterNames(bus_handle, rdkc->component_name, rdkc->dbus_path, "Device.", 0, 
-                                          &num_param_infos, &param_infos);
-    if (ccsp_err != CCSP_SUCCESS)
+    USP_LOG_Info("%s: Getting parameters and objects for CCSP components using rbus", __FUNCTION__);
```

Comment:
Log message still refers to CCSP and there's actually quite a lot of references to CCSP still in the code. For example it still includes the CCSP API header files. Please search for all occurrences of CCSP in the code and remove or fix them up.